logging-modules/log4j/src/main/java/com/baeldung/reactiveLogging/ReactiveLogging.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.reactiveLogging;
+
+import reactor.core.publisher.Flux;
+
+public class ReactiveLogging {
```

Comment:
Can you rename this to `ReactiveLoggingExample` as we have done with the other files in this folder?

Reply:
This was a mistake to leave this file - deleted.